interfaces/Portalicious/src/app/domains/registration/registration.api.service.ts

```diff
@@ -85,6 +88,61 @@ export class RegistrationApiService extends DomainApiService {
     });
   }
 
+  changeStatus({
+    projectId,
+    paginateQuery,
+    status,
+    messageData,
+    dryRun = true,
+  }: {
+    projectId: Signal<number>;
+    paginateQuery: PaginateQuery | undefined;
+    status: RegistrationStatusEnum;
+    messageData?: SendMessageData | undefined;
+    dryRun: boolean;
+  }) {
+    let body: RegistrationStatusPatchDto;
+    body = {
+      status,
+    };
```

Comment:
```suggestion
    let body: RegistrationStatusPatchDto = {
      status,
    };
```

Bit of a nitpick but not sure why it needs a separate line.